### ISSUE TITLE
Unmarshal error and panic

### ISSUE THREAD

**Issue by xXx-st4cktr4c3-xXx** [Threadfin/Threadfin#35]
`2023/01/06 12:10:04 [Threadfin] [ERROR] json: cannot unmarshal object into Go value of type []*src.GithubReleaseInfo () - EC: 0`

```
* * * * * FATAL ERROR * * * * *
OS:   linux
Arch: amd64
Err:  runtime error: index out of range [0] with length 0
panic.go:838 runtime.gopanic
panic.go:89 runtime.goPanicIndex
xepg.go:217 threadfin/src.createXEPGMapping.func1
xepg.go:260 threadfin/src.createXEPGMapping
xepg.go:109 threadfin/src.buildXEPG
config.go:268 threadfin/src.StartSystem
threadfin.go:197 main.main
proc.go:259 runtime.main
asm_amd64.s:1572 runtime.goexit
* * * * * * * * * * * * * * * *
```

**Comment by Fyb3roptik** [comment]
Patched in 0.9.21 - To be released

**Comment by Fyb3roptik** [comment]
0.9.21 deployed. Closing for now. Please re-open if this issue persists.
